### ISSUE TITLE
How to get confidence?

### ISSUE THREAD

**Issue by duncare27** [TJCVRS/CRNN_Tensorflow#51]
How to get output confidence when predict object?

Thank you sincerely!

**Comment by TJCVRS** [comment]
@duncare27 You can add the softmax layer after the logits

**Comment by duncare27** [comment]
I find variable _raw_pred_ is argmx of softmax the logits. But i don't know get the confidence. I am confused about softmax the logits before predict. Please help me the code. 
Sorry my English not good

Thanks

**Comment by TJCVRS** [comment]
@duncare27 Softmax layer will ouput the confidence of the lstm output. You can get the confidence map by adding a softmax layer after net output

**Comment by vinayakkailas** [comment]
Can you elaborate on this, "You can get the confidence map by adding a softmax layer after net output"

**Comment by wiistriker** [comment]
@vinayakkailas did you able to get confidence for each character?
